src/core/column/sentinel_fw.cc

```diff
@@ -139,11 +139,8 @@ size_t SentinelFw_ColumnImpl<T>::memory_footprint() const noexcept {
 // Data access
 //------------------------------------------------------------------------------
 
-#if DT_COMPILER_MSVC
-  #pragma warning(push)
-  // unreachable code
-  #pragma warning(disable : 4702)
-#endif
+DISABLE_MSVC_WARNING(4702)
```

Comment:
May be don't remove the comment line `// unreachable code`, so that we know what warning it is.

Reply:
Oh, right. I didn't realize that comment was describing the warning.